src/ansible_compat/config.py

```diff
@@ -52,12 +52,12 @@ def parse_ansible_version(stdout: str) -> Tuple[str, Optional[str]]:
         r"^ansible \[(?:core|base) (?P<version>[^\]]+)\]", stdout, re.MULTILINE
     )
     if match:
-        return match.group("version"), None
+        return Version(match.group("version"))
     # ansible-base 2.10 and Ansible 2.9: 'ansible 2.x.y'
     match = re.search(r"^ansible (?P<version>[^\s]+)", stdout, re.MULTILINE)
     if match:
-        return match.group("version"), None
-    return "", "FATAL: Unable parse ansible cli version: %s" % stdout
+        return Version(match.group("version"))
+    raise AnsibleCompatError("Unable to parse ansible cli version: %s" % stdout)
```

Comment:
Compat is too generic. It should be specific inheriting `LookupError`

Reply:
How about `InvalidPrerequisiteError` which already exists?